pom.xml

```diff
@@ -60,6 +60,11 @@
             <version>1.9</version>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.3.3</version>
```

Comment:
This is very old... Jackson is now up to 2.7, any reason for not using the latest?